src/wallet/bdb.cpp

```diff
@@ -716,7 +716,7 @@ bool BerkeleyBatch::StartCursor()
     assert(!m_cursor);
     if (!pdb)
         return false;
-    int ret = pdb->cursor(nullptr, &m_cursor, 0);
+    int ret = pdb->cursor(activeTxn, &m_cursor, 0);
```

Comment:
Is this a bugfix?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18608#discussion_r453369905

> Is this a bugfix?

It's needed to avoid errors if a transaction and a cursor are active at the same time. This didn't used to happen so it wasn't really a bug, but this change is more correct, and necessary for the new erase function.